mysql-test/lib/mtr_cases.pm

```diff
@@ -639,7 +639,7 @@ sub optimize_cases {
     # Check that engine selected by
     # --default-storage-engine=<engine> is supported
     # =======================================================
-    my %builtin_engines = ('myisam' => 1, 'memory' => 1, 'csv' => 1);
+    my %builtin_engines = ('myisam' => 1, 'memory' => 1, 'csv' => 1, 'rocksdb' => 1);
```

Comment:
I do not think those changes have any impact on this issue. All rocks db mtr tests are written with the basic assumption that rocksdb is the default storage engine. In facebooksql, it is compiled in and configured as such, not so in Percona Server. So this proposed change, running mtr with `--mysqld=--default-storage-engine=rocksdb --mysqld=--default-tmp-storage-engine=rocksdb` (or any other non-builtin engine) would result in all tests skipping with `[ skipped ]  'rocksdb' not supported`. Similarly, doing the same thing with TokuDB results in the same issue. I would certainly like a better solution than this but other than re-writing all of the tests to be more explicit in specifying the default engine or using `... ENGINE=ROCKSDB` I can't see any other way to fix it.